mako_config/mobile-bert.config

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+benchmark_key: "5538704950034432"
+benchmark_name: "MobileBert (Pixel 4) (Time Unit: ms)"
+project_name: "iree"
+
+# Only owners can write to the benchmark
+owner_list: "hanchung@google.com"
+owner_list: "buildkite-agent@iree-oss.iam.gserviceaccount.com"
+
+# Define the name and type for x-axis of run charts
+input_value_info: {
+  value_key: "t"
+  label: "time"
+  type: TIMESTAMP
+}
```

Comment:
I think we probably don't want a timestamp here? We should be tagging this by commit

Reply:
This is for a run chart, not benchmark chart. E.g., https://mako.dev/run?run_key=5693387618385920&~c=1&~v=1&~g=1

For now, we only have one number which indicates the total execution time. I have been thinking more about this, we probably can make it be a profiling plot, so when click into a run, we will see a plot. Outside a run, we can aggregate the numbers (or just use the last one) and show in benchmark chart (https://mako.dev/benchmark?benchmark_key=5538704950034432)

In both cases, it is indeed TIMESTAMP.

The commit is attached as a "Hover Text" to a run, which is showed up in benchmark chart (when you move the curser on the data point).

Reply:
Got you, this is great and I totally agree. Let's use `NUMERIC` now, and we can iterate on it later when we want to do more stuff. I actually plan to have more repetitions in one run, so we can make numbers more stable. `NUMERIC` makes sense in any cases.